packages/devtools_app/lib/src/debugger/flutter/console.dart

```diff
@@ -87,7 +87,7 @@ class _ConsoleState extends State<Console> {
   void _scrollToBottom() async {
     await scrollController.animateTo(
       scrollController.position.maxScrollExtent,
-      duration: const Duration(milliseconds: 100),
+      duration: shortDuration * 2,
```

Comment:
can you add a new duration between the short and defaultDuration that is 100ms?

Reply:
will do

Reply:
(`rapidDuration`, for want of a better name)